package.json

```diff
@@ -44,6 +44,9 @@
     "autoprefixer": "6.7.7",
     "axe-core": "2.0.7",
     "axe-webdriverjs": "0.4.0",
+    "babel-core": "6",
```

Comment:
I see there's a yarn.lock file added. Does the signin-widget use yarn or npm cli?